Makefile

```diff
@@ -42,6 +42,9 @@ doc-build: check_venv
 	type sphinx-build || { echo "please run `make install-docs` to build docs"; false; }
 	make -C docs html
 
+doc-preview: check_venv doc-build
+	python3 -m http.server --directory docs/_build/html/
+
```

Comment:
We could also consider https://pypi.org/project/sphinx-autobuild/. (I've never used it, and just learned about its existence, so maybe that's for a separate PR.)

Reply:
> We could also consider https://pypi.org/project/sphinx-autobuild/.

Oooo, nice find! TIL -- if no one else does, I'll try to integrate that this afternoon PT (I'm only doing 1/2 day today). If it gets messy, I'll land this as-is.

> I've taken the freedom to fix some links in the README file on this branch,

Much appreciated! IMO, PRs are like a wiki -- no permission to modify needed!